modules/etcd-tuning-parameters.adoc

```diff
@@ -0,0 +1,137 @@
++
+[source, terminal]
+----
+$ oc describe etcd/cluster | grep "Control Plane Hardware Speed"
+----
++
+.Example output
+[source,terminal]
+----
+Control Plane Hardware Speed:  ""
+----
+
+. Force an etcd rollout, which restarts all etcd pods one at a time to pick up the new values, by entering the following command:
++
+[source,terminal]
+----
+$ oc patch etcd/cluster --type=merge -p '{"spec": {"forceRedeploymentReason": "hardwareSpeedChange-'"$( date --rfc-3339=ns )"'"}}'
+----
++
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
+----
+
+. Wait for etcd pods to roll out:
++
+[source,terminal]
+----
+oc get pods -n openshift-etcd -w
+----
++
+The following output shows the expected entries for master-0. Before you continue, wait until all masters show a status of `4/4 Running`.
++
+.Example output
+[source,terminal]
+----
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Pending             0          0s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Pending             0          0s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     ContainerCreating   0          0s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     ContainerCreating   0          1s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           1/1     Running             0          2s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Completed           0          34s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Completed           0          36s
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Completed           0          36s
+etcd-guard-ci-ln-qkgs94t-72292-9clnd-master-0            0/1     Running             0          26m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Terminating         0          11m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Terminating         0          11m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Pending             0          0s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Init:1/3            0          1s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Init:2/3            0          2s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     PodInitializing     0          3s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  3/4     Running             0          4s
+etcd-guard-ci-ln-qkgs94t-72292-9clnd-master-0            1/1     Running             0          26m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  3/4     Running             0          20s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Running             0          20s
+----
+
+. Enter the following command to review to the values:
++
+[source,terminal]
+----
+$ oc describe -n openshift-etcd pod/<ETCD_PODNAME> | grep -e HEARTBEAT_INTERVAL -e ELECTION_TIMEOUT
```

Comment:
It's been suggested that we include the currently defined mapping for each of the Profiles with a note that the actual underlying values are subject to change at any time.

(I'm intentionally not including default "" because it varies depending on the platform)

Standard
ETCD_HEARTBEAT_INTERVAL = 100
ETCD_LEADER_ELECTION_TIMEOUT = 1000

Slower
ETCD_HEARTBEAT_INTERVAL = 500
ETCD_LEADER_ELECTION_TIMEOUT = 2500

Reply:
@dusk125 I added a table to include that info. Let me know what you think.

Reply:
Looks good thanks!